ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/MessageScrubber.java

```diff
@@ -295,6 +298,20 @@ public static void scrub(Map<String, String> attributes, ObjectNode json)
       processForBug1751955(json);
     }
 
+    if ("BrowserStack".equals(attributes.get(Attribute.ISP_NAME))) {
+      String tags = Strings.nullToEmpty(attributes.get(Attribute.X_SOURCE_TAGS));
+      List<String> tagList = Arrays.stream(tags.trim().split("\\s*,\\s*"))
+          .filter(s -> s.length() > 0).collect(Collectors.toList());
+      if (!tagList.contains("automation")) {
```

Comment:
I was at first thinking this might need to be a case insensitive match (e.g. an [example in the docs](https://github.com/mozilla/gcp-ingestion/blob/c0dabae9e73e6a0a9d4ae42a24aea516b352fd98/docs/architecture/edge_service_specification.md?plain=1#L54) has "Automation" capitalized), but it looks like the [copy deduplicate logic](https://github.com/mozilla/bigquery-etl/blob/79fc36a700d9fc0704bf35a8ec7d1b93e6705345/bigquery_etl/copy_deduplicate.py#L41-L43) this is intended to trigger uses a case sensitive comparison with "automation" in lowercase, so this is consistent with that.

Reply:
I've now updated that example as part of this PR.